lang/en.json

```diff
@@ -1,12 +1,16 @@
 {
-    "LoginLdap": {
+    "LoginLdap": {  
+        "AccessSyncSettings": "Access Synchronization Settings",
```

Comment:
Is there a reason you re-ordered the translation tokens? Can you change the order so the commit has only changes relevant to the purpose of the pull request?